pkg/README.md

```diff
@@ -0,0 +1,30 @@
+# Protobuf packaging
+
+This directory contains Bazel rules for building packaging and distribution
+artifacts.
+
+*Everything in this directory should be considered experimental.*
```

Comment:
Since we are using the java package rules to build, I'm not sure this is true.

Reply:
Yeah, I'm more writing for users who might be tempted to depend on behavior. I'll tweak to say it's internal and subject to change.